### PR TITLE
chore: @shapeshiftoss/types version references and tokens reference

### PR DESCRIPTION
**Pull request by GMSteuart**
- Reverts [removal of the tokens](https://github.com/shapeshift/lib/pull/577/commits/fd09bc2b74817c72a14fb0fff361fa38ede29a39) reference when only one reference was reverted.
- Updates @shapeshiftoss/types package references to 3.0 from the [release not updating the submodules](https://github.com/shapeshift/lib/runs/6233423908?check_suite_focus=true) that need it.

### CODE REVIEW

**Comment by GMSteuart** [comment]
@0xdef1cafe please review

**Comment by GMSteuart** [comment]
@cjthompson this will fix the typings problems occurring in build

**Review by elmutt — DISMISSED** [review]

**Comment by elmutt** [comment]
should get this merged soon for the tokens revert

**Review by 0xApotheosis — CHANGES_REQUESTED** [review]
Commit `yarn.lock` diff?

**Comment by GMSteuart** [comment]
@0xApotheosis updated heres [yarn.lock](https://github.com/shapeshift/lib/pull/585/files#diff-51e4f558fae534656963876761c95b83b6ef5da5103c4adef6768219ed76c2de)

**Comment by 0xApotheosis** [comment]
@GMSteuart merge conflict with `yarn.lock` - please merge in master, re-run `yarn`, then commit the resulting `yarn.lock`.

**Review by 0xApotheosis — APPROVED** [review]
Builds correctly.

**Review by gomesalexandre — COMMENTED** [review]
@GMSteuart Can you fix the conflict in `packages/chain-adapters/package.json` ? 🙏🏽 
Tested locally after mergefixing it and it builds fine. Will accept after the mergefix.

**Comment by GMSteuart** [comment]
@gomesalexandre should be good to go. 🙂

**Review by gomesalexandre — APPROVED** [review]
Tested again locally and build still passes without errors.

File: yarn.lock [hunk 1]
```diff
@@ -2833,7 +2833,7 @@
     google-protobuf "^3.17.0"
     long "^4.0.0"
 
-"@shapeshiftoss/types@^2.0.0", "@shapeshiftoss/types@^2.1.0":
+"@shapeshiftoss/types@^2.0.0":
```
Comment (kaladinlight) [inline yarn.lock:2836]: It seems like there is a yarn lock issue related to this types package version. We are trying to reference version ^3.0.0 from `market-service`, but there is only one version reference of `2.11.0` in the yarn lock. I would expect a v2.xx version for something like `unchained-client` that hasn't been updated along with a v3.xx version for `market-service`

Comment (kaladinlight) [reply yarn.lock:2836]: Disregard. Peer/Dev dependencies scenario...

**Review by kaladinlight — APPROVED** [review]

**Review by theoboldfrazier — APPROVED** [review]

**Pull request merged**